packages/ember-routing-htmlbars/lib/keywords/action.js

```diff
@@ -7,7 +7,7 @@ import { keyword } from 'htmlbars-runtime/hooks';
 import closureAction from 'ember-routing-htmlbars/keywords/closure-action';
 
 /**
-  The `{{action}}` helper provides a way to pass triggers for behavior (usually
+  The `{{action` helper provides a way to pass triggers for behavior (usually
```

Comment:
I use `{{action` informally in Slack, but I'd prefer `{{action}}` to `action`. Talking specifically about `action`, actions are confusing enough that the visual differentiator help people establish context. Same for "the `input` helper" vs "the `{{input}}` helper" IMO.
I still agree on always including an example even if `{{action}}` is decided on.